packages/nestjs-module/package.json

```diff
@@ -38,6 +38,7 @@
   "peerDependencies": {
     "@nestjs/common": "^7.0.0",
     "@nestjs/core": "^7.0.0",
+    "@nestjs/microservices": "^7.0.0",
```

Comment:
The module package itself shouldn't have a dependency on any of the `@nestjs/` packages other than `common` and `core`.

Reply:
I see that the reason for the dependency is the context that comes from the `@nestjs/microservices` package. Is there anything we can do to possibly make this not required? I'd rather not force users to install `@nestjs/microservices` to use the package, or have to enable `skipLibCheck` as I know that many people prefer to have that when it comes to strict mode.

Reply:
Yeah, it was only for context type. To remove it from there we could easly just create our interface for that, because Nest inject request object in ther if you use that in case of http but when you use that in case some other protocol like grpc then you get RequestContextHost, so we could create our types for that as it could be request or context and then I think we could remove that dependency from there.